toolkits/springernature/packages/springernature-user-details/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@springernature/springernature-user-details",
-  "version": "1.3.4",
+  "version": "2.0.0",
   "license": "MIT",
   "description": "Springer Nature branded logged-in user details component",
   "keywords": [],
   "author": "Springer Nature",
-  "brandContext": "^4.3.1",
+  "brandContext": "^10.0.0",
   "dependencies": {
     "@springernature/global-expander": "^2.0.2"
```

Comment:
we shouldn't be doing this. Components should not depend on other components. But leave it here for now and I will think how to tackle this.

Reply:
It was there already and it intrigued us. Glad you spotted it. So I believe I resolve the conversation on that one and I assume you have it on your todo somewhere